tests/cmdline/commands/test_export.py

```diff
@@ -213,6 +201,27 @@ def test_migrate_force(self):
                 self.assertTrue(os.path.isfile(filename_output))
                 self.assertEqual(zipfile.ZipFile(filename_output).testzip(), None)
 
+    def test_migrate_in_place(self):
+        """Test that passing the -i/--in-place option will overwrite the passed file."""
+        archive = 'export_v0.1_simple.aiida'
+        target_version = '0.2'
+        filename_input = get_archive_file(archive, filepath=self.fixture_archive)
+        filename_tmp = next(tempfile._get_candidate_names())
+
+        try:
+            # copy file (don't want to overwrite test data)
+            shutil.copy(filename_input, filename_tmp)
+
+            options = [filename_tmp, '--in-place', '--version', target_version]
+            result = self.cli_runner.invoke(cmd_export.migrate, options)
+            self.assertIsNone(result.exception)
+            self.assertTrue(os.path.isfile(filename_tmp))
+            self.assertEqual(zipfile.ZipFile(filename_tmp).testzip(), None)
```

Comment:
What is this line doing? I don't understand why you are comparing to `None`

Reply:
Good question - I copied this from the test above.
See https://docs.python.org/3/library/zipfile.html#zipfile.ZipFile.testzip